daffodil-schematron/src/test/scala/org/apache/daffodil/validation/schematron/TestBasicValidation.scala

```diff
@@ -68,7 +68,7 @@ class TestBasicValidation {
     val factory = new TransformerFactoryImpl()
     factory.setURIResolver(resolver)
     val sch = getClass.getClassLoader.getResourceAsStream("sch/schematron-1.sch")
-    val rules = Transforms.fromSch(sch, factory)
+    val rules = Transforms.from(sch, Sch, factory)
```

Comment:
Might be a separate ticket, but it seems to me we want to be able to write TDML tests with validation settings that will cause them to invoke schematron as part of the runtime. E.g., define a tdml:testSuite with defaultValidation="schematron" or a specific tdml:parserTestCase with validation="schematron"

Reply:
added DAFFODIL-2448